multiuser/api/che-multiuser-api-resource/src/main/java/org/eclipse/che/multiuser/resource/api/usage/tracker/RamResourceUsageTracker.java

```diff
@@ -81,9 +86,18 @@ public RamResourceUsageTracker(
           if (startingEnvironment != null) {
             currentlyUsedRamMB += environmentRamCalculator.calculate(startingEnvironment);
           }
+          if (SidecarToolingWorkspaceUtil.isSidecarBasedWorkspace(config.getAttributes())) {
+            LOG.warn(
+                "Sidecar based workspace `{}` is starting, memory of its plugins is not taken in "
```

Comment:
How about : ``"Memory consumption of a plugin in a sidecar-based workspace `{}` is not taken into account while the workspace is starting. The memory limits may not be applied correctly."``